chapter_recurrent-modern/gru_vn.md

```diff
@@ -32,17 +32,17 @@ For instance, there might be a transition between chapters in a book, or a trans
 In this case it would be nice to have a means of *resetting* our internal state representation.
 -->
 
-* Chúng ta có thể gặp phải tình huống mà những quan sát xuất hiện sớm có ảnh hưởng đáng kể đến việc dự doán toàn bộ những quan sát ở tương lai.
-Xét một ví dụ hơi không thực tế, trong đó ta có quan sát đầu tiên chứa giá trị tổng kiểm (_checksum_) và mục tiêu ở đây là phân biệt liệu giá trị tổng kiểm đó có đúng hay không tại cuối chuỗi.
+* Ta có thể gặp tình huống mà những quan sát xuất hiện sớm có ảnh hưởng lớn đến việc dự đoán toàn bộ những quan sát trong tương lai.
+Xét một ví dụ có chút cường điệu, trong đó quan sát đầu tiên chứa giá trị tổng kiểm (_checksum_) và mục tiêu là kiểm tra xem liệu giá trị tổng kiểm đó có đúng hay không tại cuối chuỗi.
 Trong trường hợp này, ảnh hưởng của token đầu tiên là tối quan trọng.
-Chúng ta muốn có một vài cơ chế cho việc lưu trữ những thông tin ban đầu quan trọng trong *ô nhớ*.
-Nếu không có cơ chế như vậy, ta phải gán một giá trị gradient cực lớn cho quan sát này, vì nó ảnh hưởng đến toàn bộ các quan sát tiếp theo.
-* Chúng ta có thể gặp phải tình huống mà một vài ký hiệu không chứa thông tin phù hợp.
-Ví dụ, khi phân tích một trang web, có thể sẽ có mã HTML phụ trợ không liên quan đến mục tiêu xác định thông tin được truyền tải trên trang web.
-Chúng ta sẽ muốn có một số cơ chế để *bỏ qua những ký hiệu như vậy* trong việc biểu diễn trạng thái tiềm ẩn.
-* Chúng ta có thể gặp phải tình huống trong đó tồn tại những khoảng ngắt giữa các phần của một chuỗi.
-Ví dụ, có thể sẽ có những đoạn chuyển tiếp giữa các chương của một quyển sách, hay chuyển biến giữa thị trường giá lên và thị trường giá xuống trong chứng khoán.
-Trong trường hợp này, sẽ tốt hơn nếu có một cách để *xoá* hay *đặt lại* các biểu diễn trạng thái ẩn về trạng thái ban đầu.
+Do đó ta muốn có cơ chế để lưu trữ những thông tin ban đầu quan trọng trong *ô nhớ*.
+Nếu không, ta sẽ phải gán một giá trị gradient cực lớn cho quan sát ban đầu vì nó ảnh hưởng đến toàn bộ các quan sát tiếp theo.
+* Một tình huống khác là khi một vài ký hiệu không chứa thông tin phù hợp.
+Ví dụ, khi phân tích một trang web, ta có thể gặp các mã HTML không giúp ích gì cho việc xác định thông tin được truyền tải.
+Do đó, ta cũng muốn có cơ chế để *bỏ qua những ký hiệu như vậy* trong việc biểu diễn trạng thái tiềm ẩn.
+* Ta cũng có thể gặp những khoảng ngắt giữa các phần trong một chuỗi.
+Ví dụ như những phần chuyển tiếp giữa các chương của một quyển sách, hay biến động giá cổ phiếu giữa hai thời điểm đóng cửa phiên giao dịch trước và mở cửa phiên tiếp theo. <!-- ví dụ này chuẩn nghĩa `khoảng ngắt` hơn -->
```

Comment:
Mình thấy dịch sát theo bản gốc `bull/bear market` như cũ mới truyền tải đúng hàm ý chứ nhỉ. 
Đóng cửa / mở của phiên giao dịch chỉ là ngắt quảng về mặt thời gian của dữ liệu, không cùng hàm ý với việc thị trường đang có trend tăng hay trend giảm như `bull/bear market`, nên mới cần `reset`.
Mô hình dự đoán giá cổ phiếu ngày mai mà tới cuối ngày reset lại latent vector thì còn làm ăn được gì.
```suggestion
Ví dụ như những phần chuyển tiếp giữa các chương của một quyển sách, hay chuyển biến xu hướng giữa thị trường giá lên và thị trường giá xuống trong chứng khoán.
```

Reply:
hm em thấy câu trên là `We might encounter situations where there is a logical break between parts of a sequence`, tức ý câu sau tìm ví dụ 1 khoảng ngắt quãng thời gian. Bull/bear market kiểu về dài hạn vẫn có thể coi là liên tục, ko rõ ý khoảng ngắt.
> Mô hình dự đoán giá cổ phiếu ngày mai mà tới cuối ngày reset lại latent vector thì còn làm ăn được gì.

ý này thì em nghĩ tới cuối ngày reset lại cũng đâu có vấn đề gì nhỉ, dữ liệu trong ngày đã thu thập hết rồi, có thể dự đoán giá ngày mai dựa vào dữ liệu đó. Còn thời điểm bắt đầu phiên ngày mai thì vì có khoảng ngắt quãng nên sẽ ko bắt đầu tại closing price ngày hôm trước nên việc reset lại state cuối ngày hôm trước giúp cải thiện hơn chứ?

Reply:
`logical break` chứ không phải `temporal break` nhé @cuongvng . `temporal break` có thể là `logical break`, nhưng `logical break` chưa chắc là `temporal break`.
Ví dụ như 12h trưa `Trump` tuyên bố cấm vận `TQ và Nga`, không cần chờ tới cuối ngày thì mô hình vẫn cần phải reset để cập nhật trend là giá cổ phiếu đa phần sẽ giảm mạnh (trừ mấy công ty bán vũ khí).

Reply:
em sẽ chuyển về cách dịch gốc.